README.md

```diff
@@ -1,6 +1,6 @@
 # deoplete-zsh
 
-Zsh completion for deoplete.nvim  
+Zsh completion for deoplete.nvim
```

Comment:
If delete two space, would be good also delete the "Currently, my hobby project."
In other words, This project become a not hobby project, start as a regular project.